.github/workflows/conformance-k8s-network-policies.yaml

```diff
@@ -27,8 +27,8 @@ jobs:
              -I '^[ ]\{2\}name: cilium.*' \
              -I '^Keep file in synced with.*' \
              -I '{{- if.*' \
-             cilium-agent-clusterrole.yaml \
-             cilium-preflight-clusterrole.yaml
+             cilium-agent/clusterrole.yaml \
+             cilium-preflight/clusterrole.yaml
```

Comment:
Please squash this commit into the first one (which does the renaming). This ensure that if we ever have to `git bissect` on CI tests that each commit itself is buildable.

Reply:
squashed